gcn_kafka/core.py

```diff
@@ -16,7 +16,7 @@
 
 def get_config(mode, config, **kwargs):
     # Merge configuration from user.
-    config = {**(config or {}), **kwargs}
+    config.update({k:v for (k,v) in kwargs.items() if v})
```

Comment:
1. Copy the dictionary, to make sure that it is not modified in place. (The caller might be planning to use it again!)
2. Allow merging items that are False.

```suggestion
    config = dict(config)
    config.update({k: v for k, v in kwargs.items() if v is not None})
```